src/middlewares/coopSectionsValidation.js

```diff
@@ -13,17 +13,23 @@ const coopSectionsValidation = (req, _res, next) => {
   const { sections } = req.body
   if (sections) {
     for (const section of sections) {
-      if (section.quizzes.length || section.lessons.length || section.attachments.length) {
+      if (section.activities) {
+        if (Array.isArray(section.activities)) {
```

Comment:
it seems to me that it's possible to remove one level of nesting

Reply:
yeah, thanks